motor-control/tests/test_motor_pulse.cpp

```diff
@@ -1,13 +1,23 @@
 #include "catch2/catch.hpp"
+#include "common/tests/mock_message_queue.hpp"
 #include "motor-control/core/motor_interrupt_handler.hpp"
-#include "pipettes/tests/mock_message_queue.hpp"
+#include "motor-control/tests/mock_motor_hardware.hpp"
 
 using namespace motor_handler;
 
 #define TO_RADIX 31
+
 static constexpr sq0_31 default_velocity =
     0x1 << (TO_RADIX - 1);  // half a step per tick
 
+#define BUILD_INTERRUPT_HANDLER_Q(name, queuename, cqueuename) \
```

Comment:
Are my eyes deceiving me? Is that a macro?? Nice to see that the oldies still have their place in our code base.

Reply:
A classic of the genre, really. Probably should have just made it an object but I also kind of missed doing macros and it's just one file

Reply:
I was under the impression we didn't like macros

Reply:
I can change it. We don't like macros but the big reason we don't like them is their maintainability - and one that's scoped to precisely one test file isn't really that bad IMO.

Reply:
Sorry for starting this. I wasn't really criticizing. Just thought it was funny to see a macro.